.github/workflows/run-tests.yml

```diff
@@ -20,8 +20,18 @@ jobs:
       - run: npm install
       - run: npm run check-format
       - run: make web_unit
-      - run: make web_integration
+      - name: make web_integration
+        run: |
+          cd integration_tests_server
+          npm install
+          node index.js &
+          cd ..
+          make web_integration
         env:
+          INTEGRATION_TESTS_APP_MT1_APP_ID: ${{ secrets.INTEGRATION_TESTS_APP_MT1_APP_ID }}
+          INTEGRATION_TESTS_APP_MT1_SECRET: ${{ secrets.INTEGRATION_TESTS_APP_MT1_SECRET }}
+          INTEGRATION_TESTS_APP_EU_APP_ID: ${{ secrets.INTEGRATION_TESTS_APP_EU_APP_ID }}
+          INTEGRATION_TESTS_APP_EU_SECRET: ${{ secrets.INTEGRATION_TESTS_APP_EU_SECRET }}
```

Comment:
Wondering which account those apps belong to. No changes are needed in this PR. It's just an operational comment to know which secrets will be stores for these.

Reply:
I'll add something to the engineering manual